### ISSUE TITLE
Problem installing plugin -> pfSense doesn't have plugin

### ISSUE THREAD

**Issue by andreapx** [VictorRobellini/pfSense-Dashboard#65]
Hi,
in the documentation i read:
_Perhaps the easiest method of installing these plugins is by utlizing the "Filer" plugin in pfSense_
but pfSense doesnt have plugins, it has a package manager (or I really can't find the plugin section).
I've tried searching in the package manager, but it doesn't find anything.
What am I doing wrong?
Thanks!

**Comment by tuxthepenguin84** [comment]
I'm running pfSense 2.6.0 Community Edition
Go to System -> Package Manager -> Available Packages, scroll down to Filer

**Comment by vasquezmi** [comment]
Correct....the Filer plugin should be listed under available plugins.  Latest version from sysutils is 0.60.6_8.
